webpack.config.base.js

```diff
@@ -45,4 +40,15 @@ module.exports = {
   node: {
     fs: "empty",
   },
+  stats: {
+    all: false,
+    assets: true,
+    errors: true,
+    warnings: true,
+  },
+  optimization: {
+    splitChunks: {
+      chunks: "all",
+    },
+  },
```

Comment:
This is where most of the substantial changes are, The rest of the PR is updating lodash usage for bundle benefits.